cciskel-duffy

```diff
@@ -124,7 +124,7 @@ def write_inventory(args, allocated_class, topo_data=None):
             n_allocated = 0
             for machine in allocated_class:
                 if machine['job'] != '':
-                    continue
+                    pass
```

Comment:
The continue jumps out of the loop for some reason. I put the pass in place and it did the right thing (tm). I'm open to suggestions here.